test/UnitTest.cs

```diff
@@ -1884,9 +1884,9 @@ public void TumbleWindow()
             TU.ExecuteSQL(sql, "2;2;1", out phyplan);
             sql = "select tumble_start(a0, interval '10' second), tumble_end(a0, interval '10' second), " +
                 "count(*) from ast group by tumble(a0, interval '10' second)";
-            TU.ExecuteSQL(sql, "5/12/2020 7:22:10 AM,5/12/2020 7:22:20 AM,2;" +
```

Comment:
The time format is an issue. On my side the time format is different from the one in the test.

Reply:
Which part of the code controls the time formatting?

Reply:
It may come from the system environment setting - I am using windows box. One way we can enforce some systems settings at start (say put with Catalog.Init()).

Reply:
The issue is fixed by constructing a DateTime instance rather than a string literal.